app/controllers/docker_manager/admin_controller.rb

```diff
@@ -21,6 +21,13 @@ def repos
           official: Plugin::Metadata::OFFICIAL_PLUGINS.include?(r.name),
         }
 
+        plugin = Discourse.visible_plugins.find { |p| p.path == "#{r.path}/plugin.rb" }
```

Comment:
Need to be careful with this...the page errors completely on the client if you give it a wrong path:

![2024-05-03_11-46](https://github.com/discourse/docker_manager/assets/920448/840fe5dd-c8c7-4856-8c5e-38bb37681a21)

Reply:
Or maybe this is an existing problem -- either way we should account for this.

Reply:
This is now fixed by the commit https://github.com/discourse/docker_manager/pull/214/commits/8449511290707653d7145f8c0246d22e86914f12